Managers/LuaMan.cpp

```diff
@@ -762,10 +789,13 @@ int LuaMan::Create() {
             // Free function bound as member function to emulate default variables
             .def("GibThis", &GibThis)
             .def("MoveOutOfTerrain", &MOSRotating::MoveOutOfTerrain)
-            .def("ApplyForces", &MOSRotating::ApplyForces)
-            .def("ApplyImpulses", &MOSRotating::ApplyImpulses)
+			.def("GetGibWoundLimit", (int (MOSRotating:: *)() const) &MOSRotating::GetGibWoundLimit)
+            .def("GetGibWoundLimit", (int (MOSRotating:: *)(bool includeAttachablesWithAPositiveDamageMultiplier, bool includeAttachablesWithANegativeDamageMultiplier, bool includeAttachablesWithNoDamageMultiplier) const) &MOSRotating::GetGibWoundLimit)
```

Comment:
You don't need to include the identifiers it will work with just `(bool, bool, bool)` and will actually be easier to figure out since it won't span across 5 screens.

Reply:
Yeah, I generally prefer to write them out so I can more easily compare, but with these gigantor fucking names you're right that it's easier to not. I'll change that.